Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+                }
+                catch (Exception ex)
+                {
+                    Reporter.ToLog(eLogLevel.ERROR, $"Method - {MethodBase.GetCurrentMethod().Name}, Error - {ex.Message}", ex);
+                    //failed to re order the activities to match the tc steps order, not worth breaking the import because of this
+                }
+            }
+            return;
+        }
+
+        private static string StripHTML(string HTMLText, bool toDecodeHTML = true)
+        {
+            try
+            {
+                HTMLText = HTMLText.Replace("<br />", Environment.NewLine);
+                Regex reg = new Regex("<[^>]+>", RegexOptions.IgnoreCase);
+                var stripped = reg.Replace(HTMLText, "");
+                if (toDecodeHTML)
+                    stripped = HttpUtility.HtmlDecode(stripped);
+
+                stripped = stripped.TrimStart(new char[] { '\r', '\n' });
+                stripped = stripped.TrimEnd(new char[] { '\r', '\n' });
+
+                return stripped;
+            }
+            catch(Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Error occurred while stripping the HTML from QC TC Step Description/Expected", ex);
+                return HTMLText;
+            }
+        }
+
+        private static void GetStepParameters(string stepText, ref List<string> stepParamsList)
+        {
+            try
+            {
+                MatchCollection stepParams = Regex.Matches(stepText, @"\<<<([^>]*)\>>>");
+
+                foreach (var param in stepParams)
+                {
+                    string strParam = param.ToString().TrimStart(new char[] {'<'});
+                    strParam = strParam.TrimEnd(new char[] { '>' });
+                    stepParamsList.Add(strParam);
+                }
+            }
+            catch (Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Error occurred while pulling the parameters names from QC TC Step Description/Expected", ex);
+            }
+        }
+
+        public static QtestTest GetQtestTest(string testID)
+        {
+           
+            return new QtestTest();
+        }
+
+        public static ObservableList<ExternalItemFieldBase> GetALMItemFields(ALM_Common.DataContracts.ResourceType resourceType)
+        {
+            ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
+
+            //if (QCRestAPIConnect.QcRestClient == null)
+            //{
+            //    string qcbin = "qcbin";
+            //    QCRestAPIConnect.QcRestClient = new QCClient(ALMCore.AlmConfig.ALMServerURL.TrimEnd(qcbin.ToCharArray()), ALMCore.AlmConfig.ALMUserName, ALMCore.AlmConfig.ALMPassword, ALMCore.AlmConfig.ALMDomain, ALMCore.AlmConfig.ALMProjectName, 12);
+            //}
+
+            //if (QCRestAPIConnect.QcRestClient.Login())
+            {
+                if (resourceType == ALM_Common.DataContracts.ResourceType.ALL)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)